sentinel-core/src/main/java/com/alibaba/csp/sentinel/node/ClusterNode.java

```diff
@@ -47,7 +47,7 @@
      * become. so we don't concurrent map but a lock. as this lock only happens
      * at the very beginning while concurrent map will hold the lock all the time.
      */
-    private Map<String, StatisticNode> originCountMap = new HashMap<String, StatisticNode>();
+    private static volatile Map<String, StatisticNode> originCountMap = new HashMap<String, StatisticNode>();
```

Comment:
The `originMap` should not be static because we need to record statistics separated by different origins per resource. Be sure to understand the relations between the four kind of `Node`.

Reply:
嗯，你说的对。 orginNode是按照每个资源调用app维度进行统计，clusterNode是按照资源维度进行统计，entranceNode是从上下文的维度进行统计，那么可以利用entraceNode来实现把熔断降级的资源进行分组，然后同一分组的某一个资源达到阈值，则同一上下文的资源都触发熔断。 这么实现可以否？